source/renderer/app/components/widgets/Dialog.scss

```diff
@@ -38,7 +38,7 @@
 
     &::-webkit-scrollbar-track {
       margin-bottom: 12px;
-      margin-top: -8px;
+      margin-top: 12px;
```

Comment:
@aleksandardjordjeviciohk seems like we do have it here? Why do we then need to specify it in the globals in the component CSS files?

Reply:
We use now 12px in the main Dialog scss, and just override in the components scss in 2/3 dialogs when its needed.